_hackbars/bislam.md

```diff
@@ -0,0 +1,12 @@
+layout: hackbar
+title: Hey
+author: Bislam S
+---
+
+<!-- Let's become a better developers so we can figure to stop our planet from polution-->
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```

Comment:
you need to add your __MESSAGE__ on climate change

Reply:
<!-- Let's become a better developers so we can figure to stop our planet from polution-->